manager/controllers/app/select_modules.go

```diff
@@ -126,7 +127,7 @@ func (p *PlotterGenerator) validate(item *DataInfo, solution Solution, appContex
 				element.StorageAccountRegion = region
 			}
 			if element.StorageAccountRegion == "" {
-				p.Log.V(0).Info("Could not find a storage account, aborting data path construction")
+				p.Log.Fatal().Str(logging.DATASETID, item.Context.DataSetID).Msg("Could not find a storage account, aborting data path construction")
```

Comment:
Debug, not Fatal. It's a step where some of the data paths are invalidated. If copy can not be done, and is not required either, it is not a mistake to skip a data path with copy.

Reply:
OK